site/content/entry/yuzu-progress-report-aug-2022/index.md

```diff
@@ -0,0 +1,334 @@
+	" "
+	"./aboutbug.png"
+	"./aboutfix.png"
+>}}
+
+Users complained that the Control profiles drop down list had a too short character limit, not allowing them to be creative.
+german77 stepped up and {{< gh-hovercard "8783" "increased the character limit." >}}
+
+## Input improvements
+
+UI navigation with a controller is relatively recent in yuzu, so there is room for improvement.
+An example reported by users is that if asking for the user profile at boot is enabled, the controller will be able to navigate the options and start the game, but it won’t be able to actually change the profile, or cancel the boot process.
+Newcomer [jonesyUK](https://github.com/jonesyUK) {{< gh-hovercard "8716" "solves both issues" >}} with a bit of Qt magic.
+Thank you!
+
+Some users prefer to depend on ds4windows instead of yuzu’s native support for game controllers.
+While there is a good reason for that, it’s a simple way to have a single app handle XInput emulation, it can also lead to some interesting issues german77 discovered.
+For example, current versions of ds4windows can’t take two sequent vibration commands, producing a (depending on the person you ask) not very comfortable constant vibration.
+To solve this brrrrrracing issue, german77 {{< gh-hovercard "8722" "added a small delay" >}} so the app doesn’t skip the stop signal.
+
+On a more colourful topic, and with the intention of fixing a bug with `Mario Party Superstars`, german77 spent many hours cooking {{< gh-hovercard "8724" "how to properly emulate controller colours," >}} with a side dish of fixing battery level icons too!
+
+{{< imgs
+	"./colour.png| Art Attack!"
+  >}}
+
+Go to `Emulation > Configure… > Controls > Advanced` and go to town!
+
+And to close up, an oopsie from our side.
+With the initial implementation of IR camera support, the output was saved to storage, every single video frame as an individual image file…
+This led to TONS of files being written to disk, making navigation with the file manager a pain, and eating a lot of free storage. Oops.
+german77 acted quickly and {{< gh-hovercard "8805" "moved the output to a buffer." >}}
+
+If any user suffered from this and can’t remove the files, open a cmd window/terminal, navigate to the Pictures folder, and run `del IMG_00*.jpg` to wipe them all out in a single blow.
+
+## Hardware section
+
+#### Intel ARC joins the fun
+
+We got information that early adopters of A380 Intel GPUs had issues running their games using Vulkan.
+Since this hardware is not globally available yet, it’s not easy for our developers and testers to get their hands on, so please [post issues in our GitHub](https://github.com/yuzu-emu/yuzu/issues) including [log files](https://yuzu-emu.org/help/reference/log-files/), or contact us in the [forums](https://community.citra-emu.org/c/yuzu-support/14) or our [Discord server](https://discord.gg/u77vRWY).
+We’ll ask for more information if needed to find the reason for this.
+Hurray to the early adopter tax.
```

Comment:
```suggestion
We received information that early adopters of A380 Intel GPUs had issues running their games using Vulkan.
Since this hardware is not globally available yet, it’s not easy for our developers and testers to get their hands on it, so please [post issues in our GitHub](https://github.com/yuzu-emu/yuzu/issues) including [log files](https://yuzu-emu.org/help/reference/log-files/), or contact us on our [forums](https://community.citra-emu.org/c/yuzu-support/14) or our [Discord server](https://discord.gg/u77vRWY).
We’ll ask for more information, if needed, to find the reason for this.
Hurray for the early adopter woes.
```